tests/Feature/Controllers/Persons/PersonAddressControllerTest.php

```
@@ -0,0 +1,226 @@
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+use PHPUnit\Framework\Attributes\Test;
+use Tests\TestCase;
+
+class PersonAddressControllerTest extends TestCase
+{
+    use DatabaseTransactions;
+
+    #[Test]
+    public function it_displays_the_new_address_form(): void
+    {
+        Carbon::setTestNow(Carbon::parse('2025-03-17 10:00:00'));
```

PHPMD (2.15.0)

Avoid using static access to class '\Carbon\Carbon' in method 'it_displays_the_new_address_form'. (Clean Code Rules)

(StaticAccess)